components/camel-minio/src/test/java/org/apache/camel/component/minio/MinioTestUtils.java

```diff
@@ -37,17 +37,4 @@ public static Properties loadMinioPropertiesFile() throws IOException {
 
         return properties;
     }
-
-    static Properties loadMinioAccessFromJvmEnv() throws Exception {
```

Comment:
Is there a specific reason to remove this?

Reply:
it is not used. Do you want keep it ?